client/src/components/QA/AddingForm.jsx

```diff
@@ -107,7 +117,7 @@ class AddingForm extends React.Component {
               </tr>
               <tr>
                 <td></td>
-                <td className='warning_text'>For privacy reasons, do not use your full name or email address</td>
+                <td className='QA_warning_text'>For privacy reasons, do not use your full name or email address</td>
```

Comment:
I like that you're using QA abbreviation before the class name, however, I believe we are to use train-case lowercase for CSS styling. Please see this link: https://github.com/rpp29-fec-gouda/atelier/blob/main/docs/web-style-guide.md